util.py

```diff
@@ -47,6 +47,47 @@ def is_email_in_domains(email: str, domain_list: List[str]) -> bool:
     return False
 
 
+def unicode_secure_filename(filename):
+    """
+    Secure filename handling
+    Based on werkzeug secure_filename but allows unicode characters
+
+    :param filename: Filename of file to check
+
+    :returns: Filename with bad characters removed
+    """
+    # Blacklist of characters to strip
+    # No control characters, no delete, no slashes. Otherwise all other characters ok
+    remove_chars_re = compile('[\u0000-\u001F\u007F\\\/]')
```

Comment:
With this code, I don't support filenames with slashes. Do our users upload files with slashes?